server/src/concepts/category/controllers/index.js

```diff
@@ -0,0 +1,26 @@
+import addNewCategory from "../useCases/addNewCategory.js";
+import validateCreateCategory from "../model/CategoryValidation.js";
+import { getAllCategories } from "../repositories/queries.js";
+
+export const createCategory = async (req, res) => {
+  const validationCategory = validateCreateCategory(req.body);
+  if (validationCategory.error) {
+    return res.status(400).send("Invalid data");
+  }
+  try {
+    await addNewCategory(req.body);
+    return res.status(200).send(`Category created`);
+  } catch (error) {
+    return res.status(500).send(error.message);
+  }
+};
+
+export const getCategories = async (req, res) => {
+  try {
+    const allCategories = await getAllCategories();
+    if (!allCategories) return res.status(404).send("No category found!");
```

Comment:
```suggestion
    if (!allCategories) return res.status(404).send("No category found!");
```

This if is unneccessary I think, firstly, this statement always return false because allCategories would be an empty array, if there arent categories, which is true in JS. Second thing is that it is okay to return empty array into client from backend when there is no data when we try to fetch all of them.